File: speed-comparison/time_check.cpp
#include<iostream>
#include<string>
#include<ctime>

int main()
{
    auto start_time = clock();

    std::string str;

    for(unsigned i = 0 ; i < 100000; ++i)
    {
        str += std::to_string(i);
        str.push_back('\n');
    }

    puts(str.c_str());
    std::cout << "Time taken= " << clock() - start_time << "ms";
}

File: speed-comparison/time_check.java

public class time_check {
    public static void main(String[] args) {
        long startTime = System.nanoTime();

        for(int i = 0 ; i < 10000 ; ++i){
            System.out.println(i);
        }

        long endTime = System.nanoTime();
        long elapsedTime = endTime - startTime;

        System.out.println("Time taken by Java: " + ((double)elapsedTime/1000000000) + " seconds");
    }
}